app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(person["name"], person["age"])
+
+    for person_data in people:
+        person = Person.people[person_data["name"]]
+        if "wife" in person_data and person_data["wife"]:
+            person.wife = Person.people[person_data["wife"]]
+        elif "husband" in person_data and person_data["husband"]:
+            person.husband = Person.people[person_data["husband"]]
```

Comment:
The code assumes that the spouse (wife or husband) will always be present in the `Person.people` dictionary. This could lead to a `KeyError` if the spouse's name is not found. Consider adding a check to ensure the spouse exists in the dictionary before assigning.